quiz/docker-compose.prod.yaml

```diff
@@ -0,0 +1,48 @@
+services:
+  proxy:
+    build:
+      context: ./nginx
+    ports:
+      - 8081:80
```

Comment:
Nie odpalałbym proxy na porcie 8081, szczególnie dla proda tylko 
80 bez SSL
443 z SSL, ale to dopiero jak sie ogarnie certy